### PR TITLE
Added Touch support to the calls

### PR DESCRIPTION
**Pull request by sharinganthief**

### CODE REVIEW

**Comment by sean-public** [comment]
On most mobile browsers, these would fire twice; once for `touchstart` and once for `click`, for example. I guess you wouldn't notice because the event handlers are safe to re-run as written.

I think we can play it safe by adding `e.stopPropagation()` and `e.preventDefault()`, as discussed in this brief post: https://joshtronic.com/2015/04/19/handling-click-and-touch-events-on-the-same-element/

**Comment by sharinganthief** [comment]
Updated with event termination calls

File: src/js/main.js [hunk 1]
```diff
@@ -31,13 +31,27 @@ function showPopup(){
 			$r.clone().attr('id', '_rf_highlight').prependTo('body').append(controls).fadeIn(500);
 
 			// handle the two new buttons we attached to the popup
-			$('#_rf_closebtn').click(hidePopup);
-			$('#_rf_disablebtn').click(function(b){
+			$('#_rf_closebtn').on('click touchstart', function() { 
+				
+				e.stopPropagation();
```
Comment (sean-public) [inline src/js/main.js:36]: Where does `e` come from here? Should it be the first arg to the function?

**Comment by sean-public** [comment]
After testing on mobile, this doesn't seem necessary. The 300ms delay between the events firing feels fine since so much precedent there on the web.

**Pull request closed without merging**